test/helper.rb

```diff
@@ -1,3 +1,9 @@
+require "simplecov"
+SimpleCov.start do
+  add_filter "/test/"
+  add_filter "/vendor/"
```

Comment:
Probably don't need the `/vendor/` filter. I don't see that directory in this repo.